### DISCUSSION TITLE
List actions

### DISCUSSION THREAD

**Discussion by OldManDanky** [scandum/tintin#160] [Script help]
I am looking to do something like the following:

#list {people} {bob|mary|sue}

then:

Create an alias to do something like:

slap $list, in a 3 second interval.

Probably very simple, but I'm old, and brain no work good.

**Comment by scandum** [comment]
It's a bit convoluted, but the following should work:
```
#var {people} {bob;mary;sue}

#alias slapall
{
   #var delay 0;
   #foreach {$people} {person}
   {
      #line sub var #delay $delay slap $person;
      #math delay $delay + 3
   }
}
```

**Comment by OldManDanky** [comment]
Scandum,

 

Thank you, this helped me!
